scripts/check-versions.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+archs="amd64 arm64 s390x"
+
+for release in $(cat versions.txt); do
+  for arch in $archs; do
+    RESP=$(curl -L -o /dev/null --silent -Iw '%{http_code}' "https://dl.k8s.io/release/${release}/bin/linux/${arch}/kubectl")
+    if [ "$RESP" != "200" ]; then
+      echo "Did not get a 200 response from 'https://dl.k8s.io/release/${release}/bin/linux/${arch}/kubectl'"
```

Comment:
For easier debugging:
```suggestion
    if [ "${RESP}" != "200" ]; then
      echo "Error: Got a ${RESP} response for 'https://dl.k8s.io/release/${release}/bin/linux/${arch}/kubectl'"
```

Only added the `Error: ` as we are `exit 1`.